src/App.tsx

```diff
@@ -13,10 +12,12 @@ import {
 } from "@arbitrum/sdk";
 import { useNavigate, useParams, generatePath } from "react-router-dom";
 
+import { useNetwork, useSigner } from "wagmi";
 import { ethers } from "ethers";
 import {
   JsonRpcProvider,
```

Comment:
this comment is on the existing code and not sth introduced by this branch

but we should remove `JsonRpcProvider` as its only usage is for typing a variable of type `StaticJsonRpcProvider`